ibtk/src/utilities/LibMeshSystemVectors.cpp

```diff
@@ -29,48 +28,35 @@
 
 /////////////////////////////// STATIC ///////////////////////////////////////
 
-namespace
-{
-bool
-vec_stored_in_map(const std::string& vec_name)
-{
-    if (vec_name == "solution" || vec_name == "current" || vec_name == "current_local_solution") return false;
-    return true;
-}
-} // namespace
-
 /////////////////////////////// NAMESPACE ////////////////////////////////////
 
 namespace IBTK
 {
 /////////////////////////////// CLASS DEFINITION /////////////////////////////
 
-LibMeshSystemVectors::LibMeshSystemVectors(const std::vector<IBTK::FEDataManager*>& fe_data_managers,
+LibMeshSystemVectors::LibMeshSystemVectors(const std::vector<libMesh::EquationSystems*>& equation_systems,
                                            std::string system_name)
-    : d_fe_data_managers(fe_data_managers), d_system_name(std::move(system_name))
+    : d_system_name(std::move(system_name))
```

Comment:
I can't remember what I originally did but we should just switch this constructor out for a call to the other one with `std::vector<bool>(equation_systems.size(), true)` (a delegated constructor)